metisp/pymetis/src/pymetis/recipes/metis_det_dark.py

```diff
@@ -13,9 +13,9 @@
 
 
 class MetisDetDarkImpl(RawImageProcessor):
-    class InputSet(Detector2rgMixin, RawImageProcessor.InputSet):
+    class InputSet(RawImageProcessor.InputSet):
         class RawDarkInput(Detector2rgMixin, RawInput):
-            _tags = ["DARK_{det}_RAW"]
+            _tags = [r"DARK_2RG_RAW", r"DARK_GEO_RAW", r"DARK_IFU_RAW"]
```

Comment:
Why is this necessary? You made almost the opposite change in metis_det_lingain.py ..

But well, we aren't sure how to resolve this yet anyway